__tests__/unit/validators/size.test.js

```diff
@@ -274,6 +274,17 @@ describe('PR size validator', () => {
     expect(validation.validations[1].status).toBe('pass')
     expect(validation.validations[2].description).toBe('PR size for total additions + deletions is OK!')
     expect(validation.validations[2].status).toBe('pass')
+
+    settings.match = ['not_*.js']
```

Comment:
I don't think this isolate the `match` settings, could you please a test case where only `match` setting is tested? 
Here, i am not sure if this is working because of `ignore` or `match` since both are present

Reply:
Added a line below to set `ignore` to a blank array :)